### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by 1011X**
Reopening of #81 because of branch renaming.

Also added some changes mentioned by @skrlet13 (thank you!)

### CODE REVIEW

**Review by clarfonthey — DISMISSED** [review]

**Review by clarfonthey — DISMISSED** [review]

File: config/locales/es.yml [hunk 1]
```diff
@@ -145,8 +167,8 @@ es:
       search: Buscar
       shared_inbox_url: URL de bandeja compartida
       show:
-        created_reports: Reportes hechos por esta cuenta
-        targeted_reports: Reportes hechos sobre esta cuenta
+        created_reports: Denuncias hechos por esta cuenta
```
Comment (scarlet-tobar) [inline config/locales/es.yml]: "Denuncias hechas"

File: config/locales/es.yml [hunk 2]
```diff
@@ -145,8 +167,8 @@ es:
...
+        targeted_reports: Denuncias hechos sobre esta cuenta
```
Comment (scarlet-tobar) [inline config/locales/es.yml]: "Denuncias hechas"

**Review by scarlet-tobar — CHANGES_REQUESTED** [review]
Good, just a detail, it must say "Denuncias hechas", not "Denuncias hechos".

File: config/locales/es.yml (hunk 1 above)
Comment (1011X) [reply config/locales/es.yml]: Fixed!

File: config/locales/es.yml (hunk 2 above)
Comment (1011X) [reply config/locales/es.yml]: Fixed!

File: config/locales/es.yml [hunk 3]
```diff
@@ -631,25 +765,65 @@ es:
...
     other: Otros
+    posting_defaults: Condición base de publicación
+    public_timelines: Cronologías públicas
+  relationships:
+    activity: Actividad de cuenta
+    dormant: Latente
+    last_active: Última vez activo
+    most_recent: Más reciente
+    moved: Movido
+    mutual: Recíproco
```
Comment (1011X) [inline config/locales/es.yml:802]: ¿Está bien usar "recíproco" en vez de "mutuo" aquí? Tiene un significado similar y parece ser mas común.

**Comment by scarlet-tobar** [comment]
Yo diría que si.

**Review by clarfonthey — DISMISSED** [review]

**Comment by mal0ki** [comment]
Is there anything here that still needs to be addressed, otherwise I'll happily approve and merge.

**Comment by clarfonthey** [comment]
@skrlet13 Seems to have gone through everything, so I think everything is good!

**Comment by clarfonthey** [comment]
…although there seem to be some merge conflicts, so, @1011X will have to resolve those first.

**Review by mal0ki — APPROVED** [review]
I trust your judgement, and issues have been squashed

**Review by clarfonthey — APPROVED** [review]

**Pull request merged**